tests/CircleChevron.react.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import CircleChevron from '../client/components/CircleChevron';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
I noticed this configure line in each test. By having this configure in setup file and setting it in json file, you don't need to have this configure on each test. Here is the example you can look into: https://github.com/airbnb/enzyme/issues/1265